### DISCUSSION TITLE
please tell me a color you know

### DISCUSSION THREAD

**Discussion by zenizeni0918** [NAIST-SE/AttractingContributors#109] [General]
Thank you for your cooperation

https://github.com/zenizeni0918/AttractingContributors-color

**Comment by 9002taicher** [comment]
I added my favorite color to yours. Plz check my request.
And I'm grad you'll come [here](https://github.com/9002taicher/AttractingContributors).
Thank you!

**Comment by KDR121** [comment]
I added.
I'd be happy to help [here](https://github.com/KDR121/AttractingContributors), too.

**Comment by GuangxianZhu** [comment]
https://github.com/LeeHomchi/AttractingContributors_Sports
This is link. Thank you. Welcome to my channel to commit.
I like Black.

**Comment by zhangyongxin1212** [comment]
Red
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Comment by zhangyongxin1212** [comment]
https://github.com/zhangyongxin1212/AttractingContributors-Game/pulls
added and plz help me

**Comment by Gloria109** [comment]
maroon color is my fav
